limiter/config_file.go

```diff
@@ -0,0 +1,61 @@
+package limiter
+
+import (
+	"io/ioutil"
+	"os"
+	"sync"
+
+	"github.com/pkg/errors"
+)
+
+//iConfigFile interface provides abstraction for managing
```

Comment:
Why does this have *i* in front? Can it be for the type? See https://dave.cheney.net/2019/01/29/you-shouldnt-name-your-variables-after-their-types-for-the-same-reason-you-wouldnt-name-your-pets-dog-or-cat

Reply:
Renamed to more natural name for interface